### PR TITLE
Update to .netcore 3.1 for ui test project

### PR DESCRIPTION
**Pull request by clement-joye**

### CODE REVIEW

File: src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs [hunk 1]
```diff
@@ -22,6 +23,9 @@ namespace Sample.AspNetCore.SystemTests.Test.PaymentTests.Base
 {
     public abstract class PaymentTests : TestBase
     {
+        // requires using Microsoft.Extensions.Configuration;
+        private readonly IConfiguration Configuration;
```
Comment (payex-dev) [inline src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs:27]: Codacy Issue found: [Remove the unused private field 'Configuration'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5518978)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5518978&bid=18082980)

File: src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs [hunk 2]
```diff
@@ -22,6 +23,9 @@ namespace Sample.AspNetCore.SystemTests.Test.PaymentTests.Base
 {
     public abstract class PaymentTests : TestBase
     {
+        // requires using Microsoft.Extensions.Configuration;
```
Comment (payex-dev) [inline src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs:26]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5518978)

**Comment by NullableInt** [comment]
👍 💯

**Comment by NullableInt** [comment]
If this works locally we can problably merge this into `develop`. Getting MSBuild up to version 16.x has proven difficult.

**Comment by NullableInt** [comment]
@clement-joye is this ready to merge?

**Comment by clement-joye** [comment]
As far as i’m concerned yes

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5518978&bid=18082980)

File: src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs (hunk 1 above)
Comment (payex-dev) [inline src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs:27]: Codacy Issue found: [Remove the unused private field 'Configuration'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5518978)

File: src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs (hunk 2 above)
Comment (payex-dev) [inline src/Samples/Sample.AspNetCore.SystemTests/Test/PaymentTests/Base/PaymentTests.cs:26]: Codacy Issue found: [Remove this commented out code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5518978)

**Pull request merged**